app/services/sourceControl.js

```diff
@@ -38,7 +38,7 @@ export default class SourceControlService {
     }
 
     // eslint-disable-next-line prettier/prettier
-    return new Date((committer.timestamp - (committer.timezoneOffset * 60)) * 1000);
+    return new Date((committer.timestamp - committer.timezoneOffset * 60) * 1000);
```

Comment:
I'd prefer not to use the linted code - this has an eslint-disable command that was added to explicitly avoid the linter suggestion.  This is a personal preference, but I feel strongly for this line of code the explicit parentheses should be included.